### PR TITLE
chore(example): remove redundant instructions from dockerfile

### PR DESCRIPTION
**Pull request by konomae**
### Description

- Remove redundant instructions because the files already exist

### CODE REVIEW

File: examples/with-docker/apps/api/Dockerfile [hunk 1]
```diff
@@ -20,14 +20,11 @@ RUN apk add --no-cache libc6-compat
 WORKDIR /app
 
 # First install dependencies (as they change less often)
-COPY .gitignore .gitignore
```
Comment (konomae) [inline examples/with-docker/apps/api/Dockerfile:23]: - This file is not required for `yarn install`
- This file is included in `/app/out/full` for build step, therefore, no need to add here

File: examples/with-docker/apps/api/Dockerfile [hunk 2]
```diff
@@ -20,14 +20,11 @@ RUN apk add --no-cache libc6-compat
...
 COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/yarn.lock ./yarn.lock
```
Comment (konomae) [inline examples/with-docker/apps/api/Dockerfile:25]: - This file is already included in `/app/out/json`

File: examples/with-docker/apps/api/Dockerfile [hunk 3]
```diff
@@ -20,14 +20,11 @@ RUN apk add --no-cache libc6-compat
...
 RUN yarn install
 
 # Build the project and its dependencies
 COPY --from=builder /app/out/full/ .
-COPY turbo.json turbo.json
```
Comment (konomae) [inline examples/with-docker/apps/api/Dockerfile:30]: - This file is already included in `/app/out/full`

File: examples/with-docker/apps/web/Dockerfile [hunk 4]
```diff
@@ -46,9 +43,6 @@ RUN addgroup --system --gid 1001 nodejs
 RUN adduser --system --uid 1001 nextjs
 USER nextjs
 
-COPY --from=installer /app/apps/web/next.config.js .
-COPY --from=installer /app/apps/web/package.json .
```
Comment (konomae) [inline examples/with-docker/apps/web/Dockerfile:50]: - This file is already included in `/app/apps/web/.next/standalone`

**Review by anthonyshew — APPROVED** [review]
Built them both and things seem to be running smoothly. Thanks!

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 488f1f4e37d6f95719bb409f1818d9eccf7ca4e1

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 488f1f4e37d6f95719bb409f1818d9eccf7ca4e1

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Pull request merged**